x/programs/runtime/import_program.go

```diff
@@ -49,18 +58,16 @@ func NewProgramModule(r *WasmRuntime) *ImportModule {
 
 				result, err := r.CallProgram(
 					context.Background(),
-					&newInfo)
```

Comment:
I think that the preference on the go side is the parenthesis sticked at the end of the last line, should I back this off ?

Reply:
yeah I just backed this off.